extensions/lifecycle/tasks/LifecycleDeleteObjectTask.js

```diff
@@ -201,7 +201,7 @@ class LifecycleDeleteObjectTask extends BackbeatTask {
                 return done(err);
             }
             const replicationStatus = objMD.getReplicationStatus();
-            if (replicationStatus && replicationStatus !== 'COMPLETED') {
+            if (['PENDING', 'PROCESSING', 'FAILED'].includes(replicationStatus)) {
                 const error = new PendingReplicationError('object has a pending replication status');
```

Comment:
Would it add any value to update this error message to include `replicationStatus`  ?

Reply:
yes, it does. Thanks.